src/js/utils/StringUtil.js

```diff
@@ -191,6 +191,14 @@ const StringUtil = {
       return null;
     }
 
+    const renderer = new marked.Renderer();
```

Comment:
Nit: I wonder if the changes here and `marked.setOptions` need to done every time or can it be done once on import.